Packs/RunZero/Integrations/RunZero/RunZero.py

```diff
@@ -0,0 +1,525 @@
+import demistomock as demisto
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *  # noqa
+
+import urllib3
+# from typing import Dict, Any
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+
+''' CONSTANTS '''
+
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'  # ISO8601 format with UTC, default in XSOAR
+MAX_RTT = 1_000_000
+DEFAULT_LIMIT = 50
+
+''' CLIENT CLASS '''
+
+
+class Client(BaseClient):
+    """Client class to interact with the service API
+
+    This Client implements API calls, and does not contain any XSOAR logic.
+    Should only do requests and return data.
+    It inherits from BaseClient defined in CommonServer Python.
+    Most calls use _http_request() that handles proxy, SSL verification, etc.
+    For this  implementation, no special attributes defined
+    """
+
+    def asset_search(self, search_str: str = None):
+        url_suffix = f'/org/assets{search_str}' if search_str else '/org/assets'
```

Comment:
If you work with `params` dict containing the `search query` you can remove this line, as it won't be added to the search if `params` is empty.